ui/src/app/workflows/components/workflow-dag/workflow-dag.tsx

```diff
@@ -144,6 +163,24 @@ export class WorkflowDag extends React.Component<WorkflowDagProps, WorkflowDagRe
             </>
         );
     }
+
+    private static getCollapsedNodeName(parent: string, hiding: number): string {
```

Comment:
hiding -> numHidden?

Reply:
maybe extract there 4 functions to their own file?

Reply:
Sure